be/src/storage/rowset/segment_v2/parsed_page.h

```diff
@@ -87,6 +88,10 @@ class ParsedPage {
     // On error, the value of |*count| is undefined.
     virtual Status read(vectorized::Column* column, size_t* count) = 0;
 
+    virtual Status read(vectorized::Column* column, vectorized::SparseRange& range) {
+        return Status::NotSupported("ParsedPage Not Support");
```

Comment:
```suggestion
        return Status::NotSupported("Read by range Not Support");
```
better to modify the error message

Reply:
done